src/pages/Trace/AddMeasurement/AddMeasurement.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useRef, ChangeEvent } from 'react';
+import './AddMeasurement.css';
+import checkIcon from '../../../assets/check-icon.svg';
+import { renderer, pathDocument } from 'canvas/Renderer';
+import { InstructionModal } from 'components/InstructionModal/InstructionModal';
+import { Input } from 'components/Input/Input';
+import { NavButton } from 'components/NavButton/NavButton';
+import { ActionButton } from 'components/ActionButton/ActionButton';
+
+interface AddMeasurementProps {
+    setUploadedFileData: React.Dispatch<React.SetStateAction<string>>;
+}
+
+export const AddMeasurement: React.FC<AddMeasurementProps> = ({ setUploadedFileData }) => {
+    const [inputMeasurement, setInputMeasurement] = React.useState<string>("");
+    
+    // Remove class that puts canvas in the background, remove picture,
+    // and reinitialize event listeners.
+    const canvasContainerRef = useRef(document.getElementsByClassName('canvasContainer')[0]);
+    useEffect(() => {
+        canvasContainerRef.current = document.getElementsByClassName('canvasContainer')[0];
+        canvasContainerRef.current.classList.remove('canvasContainerBackground');
+        canvasContainerRef.current.appendChild(renderer.measurementInit());
```

Comment:
You don't need line 23. you can just call render.measurementInit(). (And measurementInit does not need to return an element)

Reply:
ok